functions/4dn-status/src/lambda_function.py

```diff
@@ -160,13 +168,78 @@ def filter_data(data, environment):
     "Access-Control-Allow-Origin": "*",
 }
 
+# NOTE: This will treate http://mastertest-2.xxx/ as if it were just http://mastertest.xxx/
+#       so that a cloned system will behave like its twin. That probably doesn't matter
+#       a lot but was easy to do.
+
+REFERER_REGEXP = re.compile("https?[:][/][/](data|staging|cgap|fourfront-[a-z-]*[a-z])([-]?[0-9]+)?[.].*")
+
+FOURFRONT_PROD_ENV = 'fourfront-webprod'
```

Comment:
It happens to be the bucket env. But really it's not used that way. It's just a canonical name to avoid everyone having to specify all of the many names or even to know all of the many names. And though the new name is `foufront-green`, I can see people getting confused about which is green and which blue, so I thought `webprod` more memorable. (It turns out that this would work fine if literally it were changed to `fourfront-green-or-blue` or some other token that simply matches one of `blue` or `green` or `webprod` and doesn't contain `cgap`.)